samples/WebApp/tsconfig.json

```diff
@@ -1,17 +1,15 @@
 ﻿{
     "compilerOptions": {
-        "module": "umd",
+        "module": "commonjs",
```

Comment:
I kept having issues with `umd`, and though it would be nice to support something more than what we need ourselves, it's more important that it works.